ocs_ci/ocs/constants.py

```diff
@@ -157,6 +157,7 @@
 UPI_INSTALL_SCRIPT = "upi_on_aws-install.sh"
 
 DEFAULT_CLUSTERNAME = 'ocs-storagecluster'
+DEFAULT_CLUSTERNAME_EXTERNAL_MODE = 'ocs-external-storagecluster'
 DEFAULT_BLOCKPOOL = f'{DEFAULT_CLUSTERNAME}-cephblockpool'
 METADATA_POOL = f'{DEFAULT_CLUSTERNAME}-cephfilesystem-metadata'
 DATA_POOL = f'{DEFAULT_CLUSTERNAME}-cephfilesystem-data0'
```

Comment:
@shylesh we would also need to modify these values for external mode.

Reply:
@sidhant-agrawal , Can you point me to exact line where we are consuming this, I don't see it being used OR I might be missing something.

Reply:
yes, true these might not be required during deployment, but we need a way to pass correct values while running other tier tests.

Reply:
@sidhant-agrawal shall I take this in next PR

Reply:
yes, can be taken in the next PR.